### ISSUE TITLE
How to put Password *and* Token to the same password field. 

### ISSUE THREAD

**Issue by arapov** [strongbox-password-safe/Strongbox#653]
Hello, (I haven't found other way to reach out support/helpdesk, web doesn't provide any contact)

I am using some services that requires to put both **password** and **otp_token** to the same password field: i.e. password field should look like: _Pa$$w0rt456123_, where 456123 is token. 

Is there a way to configure Strongbox to certain urls, to put the token right after the password in the very same field?

thanks,
Anton

**Comment by strongbox-mark** [comment]
Hi there, there's no way to do this at the moment. The TOTP is copied to the clipboard though, so you could fill and then paste it. Would that work for you?

**Comment by arapov** [comment]
I know about TOTP copied to clipboard. It would be nice to see Strongbox to be able to use the token the way I described. I will be happy to see an RFE out for this question. :-)

tnx.

**Comment by unicorn855** [comment]
I am curious @arapov , how is the suggested way different from what you want to achieve (apart from you having to manually paste the contents of the clipboard right after the password)?
From how I understand @strongbox-mark's suggestion, it would be what you asked for, albeit 'half-automated', wouldn't it?

**Comment by arapov** [comment]
@unicorn855 yes, what @strongbox-mark suggested is the way I use it today. If it would be possible to re-define the password field content, such as $new_pass = $pass + $otp, it would be a huge improvement for me. I remember it was possible in some other keepassxc(?) ui, back when I used Linux desktop.
